audio/portaudio/Portfile

```diff
@@ -3,8 +3,10 @@
 PortSystem          1.0
 
 name                portaudio
-version             19.20140130
-revision            2
+set major           19
+set minor           6
+set reldate         20161030
+version             ${major}.${minor}.${reldate}
 categories          audio devel
```

Comment:
This isn't the old format either. If you're going to make up your own version scheme (as opposed to using the old one) then you need to add `epoch 1`. Note that there is no obvious reason to do `major`, `minor`, etc., you're not going to ever use them again. I wouldn't bother. The odds that these guys will generate another release in the future with this particular file format seem pretty low. They haven't even updated the release in years. Keep It Simple. Don't bother generating the distname, just hard code it. It's really easier than all the work you've gone through.

Reply:
The old version is ```portaudio @19.20140130_2 (audio, devel)```?
The only thing that changed here is the .6 and the weird thing they did with zeroes, for the distname.

Reply:
The old version is 19.20140130. You are changing it to 19.6.20161030. This is not compatible with the old naming scheme. Please either name it 19.6 and set the epoch to 1 to indicate the naming scheme has changed, or, better in my opinion, name it 19.20161030.

Reply:
The old version is 19.20140130. You are changing it to 19.6.20161030. This is not compatible with the old naming scheme. Please either name it 19.6 and set the epoch to 1 to indicate the naming scheme has changed, or, better in my opinion, name it 19.20161030.